typescript/src/bitcoin.ts

```diff
@@ -637,26 +628,36 @@ export function locktimeToNumber(locktimeLE: Buffer | string): number {
 
 /**
  * Creates the output script from the BTC address.
- * @param address BTC address.
+ * @param bitcoinAddress Bitcoin address.
+ * @param bitcoinNetwork Bitcoin network.
  * @returns The un-prefixed and not prepended with length output script.
  */
-export function createOutputScriptFromAddress(address: string): Hex {
-  return Hex.from(Script.fromAddress(address).toRaw().toString("hex"))
+export function createOutputScriptFromAddress(
+  bitcoinAddress: string,
+  bitcoinNetwork: BitcoinNetwork
```

Comment:
I had to add the Bitcoin network argument. See [comment](https://github.com/keep-network/tbtc-v2/pull/707/files#r1347321811) for explanation.